scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/SubmitEditServlet.java

```diff
@@ -227,7 +230,14 @@ private void upload(HttpServletRequest request, HttpServletResponse response) {
             try {
                 LOGGER.info("editedJob that will be sent:");
                 LOGGER.info(FileUtils.readFileToString(editedJob));
-                String responseS = ((SchedulerServiceImpl) Service.get()).submitXMLFile(sessionId, editedJob);
+
+                String responseS = null;
```

Comment:
I think it's responseFromService vs the response object which must be returned by this servlet. I will modify the name